src/pfe/portal/modules/metricsService/index.js

```diff
@@ -87,6 +97,13 @@ async function removeMetricsCollectorFromJvmOptions(projectDir) {
   }
 }
 
+async function deleteBackupPomXml(projectDir) {
+  const pathToBackupPomXml = getPathToBackupPomXml(projectDir);
+  if (await fs.exists(pathToBackupPomXml)) {
+    await fs.remove(pathToBackupPomXml);
```

Comment:
No real need for the `fs.exists` check here. The `fs.remove()` function doesn't error if the file doesn't exist - https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/remove.md